### PR TITLE
UI: provision volumes with labels

### LINKED ISSUE #1891: UI: Provision volumes with labels (in the PV template)
After bootstrap, a user must create *Volume*s to provide storage for Prometheus and AlertManager. We deploy a *StorageClass* for this, but the *PersistentVolumeClaim*s for these services also use a `labelSelector` to pick the 'right' *PersistentVolume*.

It's currently not possible to specify these when creating a *Volume* through the UI. As such, one can't complete the installation using the UI only, and needs to resort to using `kubectl` (e.g. by editing and applying the example manifest shipped on the ISO), which is sub-optimal.

### PR DESCRIPTION
**Pull request by carlito-scality**
**Component**:UI

**Context**: 
- After bootstrap, a user must create Volumes to provide storage for Prometheus and AlertManager. We deploy a StorageClass for this, but the PersistentVolumeClaims for these services also use a `labelSelector` to pick the 'right' PersistentVolume.

**Summary**:

**Acceptance criteria**: 
image

Screenshot 2019-10-31 at 14 17 59

---

Closes: https://github.com/scality/metalk8s/issues/1891 https://github.com/scality/metalk8s/issues/1977

### CODE REVIEW

File: ui/src/translations/en.json [hunk 1]
```diff
@@ -124,5 +123,10 @@
   "add_solution_to_environment": "Add solution to the environment {environment}",
   "version_env": "Versions",
   "add_solution": "Add the solution",
-  "advanced_monitoring": "Advanced Monitoring"
+  "advanced_monitoring": "Advanced Monitoring",
+  "add": "Add",
+  "labels": "Labels",
+  "enter_label_name": "Enter label name…",
```
Comment (gdemonet) [inline ui/src/translations/en.json]: Why the ellipsis?

File: ui/src/translations/fr.json [hunk 2]
```diff
@@ -124,5 +123,10 @@
   "add_solution_to_environment": "Ajout d'une solution à l'environnement {environment}",
   "version_env": "Versions",
   "add_solution": "Ajout de la solution",
-  "advanced_monitoring": "Supervision Avancée"
+  "advanced_monitoring": "Supervision Avancée",
```
Comment (slaperche-scality) [inline ui/src/translations/fr.json]: `Supervision avancée` (TBC, but I don't thing the French rules require to capitalize every word in a title, unlike the English ones)

Comment (gdemonet) [reply ui/src/translations/fr.json]: Agreed, whatever the actual rules are, consistency with other messages should prevail.

File: docs/operation/volume_management/volume_creation_deletion_gui.rst [hunk 3]
```diff
@@ -47,6 +48,11 @@ To access the UI, refer to :ref:`this procedure <quickstart-services-admin-ui>`
 
     .. image:: /operation/volume_management/img/volume_created.png
 
+#. If you click the new volume listed in the **Volume list**,
+   you can get more information about this volume
```
Comment (gdemonet) [inline docs/operation/volume_management/volume_creation_deletion_gui.rst]: Could be rephrased: _If you click on any volume in the **Volume list**, you will see more information in the **Volume detail** view:_

**Review by gdemonet — CHANGES_REQUESTED** [review]
I'll let @ChengYanJin review the actual React code ;)

**Review by ChengYanJin — APPROVED** [review]
LGTM!
There are two UX maybe we can improve later:
1. In `CreateVolume` page, we should check and warn users just in case they fill the label input but forget to click on `Add` button. 
2. In `VolumeInformation page` the size of Label Table should change according to how many lines do we have.

**Comment by carlito-scality** [comment]
@bert-e approve

**Review by gdemonet — APPROVED** [review]

File: ui/src/ducks/app/volumes.js [hunk 4]
```diff
@@ -212,6 +212,7 @@ export function* createVolumes({ payload }) {
     kind: 'Volume',
     metadata: {
       name: newVolume.name,
+      labels: newVolume.labels,
```
Comment (gdemonet) [inline ui/src/ducks/app/volumes.js:215]: Oh no....... I missed this, labels were not supposed to go there, but in `spec.template.metadata.labels` :(

Comment (slaperche-scality) [reply ui/src/ducks/app/volumes.js:215]: Ha indeed, otherwise we're labeling the volume itself and not the underlying PV.

**Pull request merged**